src/main/java/kr/codesquad/library/controller/BookController.java

```diff
@@ -40,9 +40,9 @@
     @GetMapping("/category/{categoryId}")
     public ApiResult<BooksByCategoryResponse> getCategoryBooks(
```

Comment:
`Nit`
```suggestion
    public ApiResult<BooksByCategoryResponse> findCategoryBooks(
```

메소드명과 동일한 형태를 위해 추천합니다.